.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,27 +1,28 @@
-## Summary
 
-*Short description*
+# Checklist
 
-## TODO
+- [ ] Target milestone
```

Comment:
I'm not sure what to do with this. Consider:

1. User A opens PR
1. User A adds wrong milestone 0.8 and ticks off Target milestone
1. Everyone now thinks that the target milestone is agreed upon?

I don't think adding a TODO will do much more than the issues we already have: PRs get merged despite not having a target milestone. The best we can do here, is really to use the list of PRs without milestones and set them in hindsight.

Reply:
When PRs target `develop` without setting a milestone, it makes it harder to do PR archaeology. I think requiring that the milestone is set correctly when the PR gets merged is reasonable.